deploy/helm/values.yaml

```diff
@@ -340,3 +340,17 @@ prometheus:
     # (this matches the behavior of `kube-prometheus-stack` chart).
     persistentVolume:
       enabled: false
+
+swoagent:
+  # Whether the SWO Agent should be deployed as part of this chart.
+  # If not, integrations are not available.
+  enabled: true
```

Comment:
I would disable it by default for now until we have an actual usage for it in the product and we are ready for customers to use it.

Reply:
We have already a usage for it right now - you can configure it anytime. We only wont show widgets for a while.

Reply:
I think there are still some rough edges that should be solved (like https://github.com/solarwinds/swi-k8s-opentelemetry-collector/pull/233#discussion_r1147609223), before we enable it by default.